apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -260,9 +260,11 @@ const NewsItem = new GQLListSchema('NewsItem', {
             }
 
             if (operation === 'update') {
-                if (sentAt) {
+                // You can update only validBefore field if news item was already sent
+                if (sentAt && (Object.keys(resolvedData) > 1 || isEmpty(get(resolvedData, 'validBefore') ))) {
```

Comment:
You have an incorrect condition

`Object.keys(resolvedData) > 1` ==> `[el1, el2, el3] > 1` ==> `false` - It will always be false

So you can update any field together with `validBefore`

Reply:
You should check that `originalInput` only has `dv`, `sender` and `validBefore`


Something like
```suggestion
const originalInputFieldNames = Object.keys(originalInput).filter(fieldName => !['dv', 'sender'].includes(fieldName))
if (originalInputFieldNames.length !== 1 || originalInputFieldNames[0] !== 'validBefore' || isEmpty(get(resolvedData, 'validBefore')))
```

Reply:
Thanks, missed that